src/styles/_components/_tabs.scss

```diff
@@ -30,3 +30,56 @@ $tabs-height: 36px;
         }
     }
 }
+
+@mixin nav-tabs() {
+    width: 100%;
+    height: 40px;
+    background: color('background');
+    border-bottom: 1px solid color('tabs', 'border');
+    @include flex();
+
+    &__wrapper {
+        @include flex();
+        margin: 0 100px;
+
+        @include breakpoint(medium) {
+            margin: 0 10px;
+        }
+        
+        &__item {
+            @include flex();
+            @include align-items-center();
+            @include justify-content-center();
+            @include font("IntervalNextReg");
+            font-size: 14px;
+            text-transform: uppercase;
+            border-bottom: 3px solid transparent;
+
+            &:not(:last-child) {
+                margin-right: 64px;
+            }
+
+            a {
+                text-overflow: ellipsis;
```

Comment:
what's the display type of this element? to be safe since we are working with overflows we should set it to block